Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppWipeRequest/MSFT_IntuneAppWipeRequest.psm1

```diff
@@ -0,0 +1,592 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [ValidateSet('Present', 'Absent')]
+        [System.String]
+        $Ensure = 'Present',
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint,
+
+        [Parameter()]
+        [Switch]
+        $ManagedIdentity,
+
+        [Parameter()]
+        [System.String[]]
+        $AccessTokens,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Wipe,
+
+        [Parameter()]
+        [System.String]
+        $createdDateTime,
```

Comment:
These are most likely read only properties and therefore should not be handled by DSC